plugins/weather/controller.js

```diff
@@ -1,12 +1,12 @@
 function Weather($scope, $interval, $http, GeolocationService) {
 
-    var language = (typeof config.language !== 'undefined') ? config.language.substr(0, 2) : "en"
+    var language = (typeof config.general.language !== 'undefined') ? config.general.language.substr(0, 2) : "en"
     var geoposition = {}
     var weather = {}
 
     weather.get = function () {
         geoloc = geoposition;
-        var language = (typeof config.language !== 'undefined') ? config.language.substr(0, 2) : "en"
+        var language = (typeof config.general.language !== 'undefined') ? config.general.language.substr(0, 2) : "en"
```

Comment:
Yeah, I think the declaration in line 3 is enough. I was a little bit confused, but didn't removed it^^